src/localization/online_localizer/online_localizer.cpp

```diff
@@ -326,94 +291,77 @@ bool OnlineLocalizer::predExists(const Node &node) const {
-        printf("[DEBUG][OnlineLocalizer] Possible accumulated cost is smaller "
-               "than previous one. Frontier should be updated\n");
-        printf(
-            "[BINGO] If you have reached this line, please contact Olga :)\n");
-        exit(0);
-        // throw 10;
+        LOG(INFO) << "Possible accumulated cost is smaller than previous one. "
+                     "Frontier should be updated";
+        LOG(FATAL)
+            << "[BINGO] If you have reached this line, please contact Olga :)";
         // update pred; update accu_costs + update frontier.
         // assign an alternative parent (the one that came in a function) to a
         // child
-        _pred[child.quId][child.refId] = parent;
-        _accCosts[child.quId][child.refId] = poss_accCost;
-        // do not forget to update the accumulated cost for a child for
-        // estimating the priority
+        pred_[child.quId][child.refId] = parent;
+        accCosts_[child.quId][child.refId] = poss_accCost;
+        // Update the accumulated cost for a child for estimating the priority
         child.accCost = poss_accCost;
-        // create a child with higher priority ( lower accCost)
-        _frontier.push(child);
+        // Create a child with higher priority ( lower accCost)
+        frontier_.push(child);
       } else {
-        // printf("Child was visited before, but new cost is smaller\n");
-        // child.print();
+        // Child was visited before, but new cost is smaller.
       }
     } else {
       // new successor
-      _pred[child.quId][child.refId] = parent;
-      _accCosts[child.quId][child.refId] = child.idvCost + parent.accCost;
-      child.accCost = _accCosts[child.quId][child.refId];
-      _frontier.push(child);
+      pred_[child.quId][child.refId] = parent;
+      accCosts_[child.quId][child.refId] = child.idvCost + parent.accCost;
+      child.accCost = accCosts_[child.quId][child.refId];
+      frontier_.push(child);
     }
   }
 }
 
 std::vector<PathElement> OnlineLocalizer::getCurrentPath() const {
   std::vector<PathElement> path;
   bool source_reached = false;
-  Node pred = _currentBestHyp;
-  // pred.print();
+  Node pred = currentBestHyp_;
 
   while (!source_reached) {
     if (pred == SOURCE_NODE) {
       source_reached = true;
       continue;
     }
-    NodeState state = pred.idvCost > _nonMatchCost ? HIDDEN : REAL;
+    NodeState state = pred.idvCost > nonMatchingCost_ ? HIDDEN : REAL;
     PathElement pathEl(pred.quId, pred.refId, state);
     path.push_back(pathEl);
-    pred = _pred.at(pred.quId).at(pred.refId);
+    pred = pred_.at(pred.quId).at(pred.refId);
   }
   return path;
 }
 
-/** check starting from the current best node id, if there is a lot of
- * hidden
- * nodes -- meaning LOST **/
-// Check N last image matches. If 90% of them are hidden - LOST!
-
+// Check N last image matches. If `ratio` of them are hidden - LOST!
```

Comment:
Is this a comment above a doxygen comment? Why not put everything in the doxygen comment?